molgenis-metadata-manager/src/main/frontend/src/store/actions.js

```diff
@@ -103,7 +102,8 @@ export default {
    */
   [GET_ATTRIBUTE_TYPES] ({commit}: { commit: Function }) {
     api.get('/api/v2/sys_md_Attribute/meta/type').then(response => {
-      commit(SET_ATTRIBUTE_TYPES, response.enumOptions.map((type) => type.toUpperCase()))
+      // Workaround for differences between Categorical MREF enum on the server and in the REST api
+      commit(SET_ATTRIBUTE_TYPES, response.enumOptions.map(type => type === 'categoricalmref' ? 'CATEGORICAL_MREF' : type.toUpperCase()))
```

Comment:
Could you add a test case and explain the 'difference between Categorical MREF and ...'